.github/workflows/ci.yml

```diff
@@ -254,14 +308,22 @@ jobs:
           cache: 'maven'
       - name: Set up Maven settings.xml
         run: Copy-Item -Path ".github/quarkus-snapshots-mvn-settings.xml" -Destination "$env:USERPROFILE/.m2/settings.xml"
```

Comment:
considering you build your own CLI and deps are available locally and 3.26 snapshot is not in remote repo, this is unnecessary, but not causing any harm; I am not entirely sure why it was needed, probably some CLI scenarios